### DISCUSSION TITLE
How to define a single package in the workspace using `node linker hoisted`

### DISCUSSION THREAD

**Discussion by EvilMoOd** [pnpm/pnpm#6176] [Q&A]
I use pnpm to implement my workspace monorepo, but when I use eletron builder to package, I need flat structure node modules, can I use the flat structure alone for the node modules in the app project under the setting workspace, while others keep the original pnpm What about the node modules structure?
The project structure is as follows, I just want to use the flat node modules structure in the app package, the electron builder can work normally, and the others remain unchanged
![image](https://user-images.githubusercontent.com/98245255/222878991-7bdf20b6-0ab6-4259-acd4-197a019e1929.png)
pnpm-workspace.yaml
![image](https://user-images.githubusercontent.com/98245255/222879017-c78888cb-fb5b-4296-86bd-079af5545bcd.png)

**Comment by iobuhov** [comment]
But why do you need flat structure? Do you have problems with builder?

**Reply by EvilMoOd** [reply]
electron-builder need it when it build electron app

**Comment by zkochan** [comment]
No, you can only set node-linker for the whole workspace monorepo.
